### PR TITLE
Update Server.schelp

### PR DESCRIPTION
**Pull request by tiagmoraismorgado**

### CODE REVIEW

File: HelpSource/Classes/Server.schelp [hunk 1]
```diff
@@ -623,6 +623,14 @@ an optional instance of link::Classes/Condition:: used for evaluating this.
 argument:: ... args
 one or more valid OSC messages.
 
+method:: latency
+Returns a floating point variable with the specified server latency
+code::
+~myLocalServer=Server.local;
+~myLocalServer.quit.boot
+~myLocalServer.latency
+::
```
Comment (snappizz) [inline HelpSource/Classes/Server.schelp]: This example isn't very insightful. We already know the syntax for how to call instance methods, but what does this method *do*? I'd argue that we don't really need an example for this method at all.

File: HelpSource/Classes/Server.schelp [hunk 2]
```diff
@@ -623,6 +623,14 @@ an optional instance of link::Classes/Condition:: used for evaluating this.
 argument:: ... args
 one or more valid OSC messages.
 
+method:: latency
+Returns a floating point variable with the specified server latency
```
Comment (telephon) [inline HelpSource/Classes/Server.schelp]: Also, it doesn't return a "variable", but a value.

File: HelpSource/Classes/Server.schelp [hunk 3]
```diff
@@ -624,7 +624,7 @@ argument:: ... args
 one or more valid OSC messages.
 
 method:: latency
-Returns a floating point variable with the specified server latency. 
+Returns a Value with the specified server latency.
```
Comment (telephon) [inline HelpSource/Classes/Server.schelp]: small "v"

File: HelpSource/Classes/Server.schelp [hunk 4]
```diff
@@ -623,6 +623,12 @@ an optional instance of link::Classes/Condition:: used for evaluating this.
 argument:: ... args
 one or more valid OSC messages.
 
+method:: latency
+Returns a Value with the specified server latency. 
+Supercollider server latency may affect supercollider behaviour in several ways. 
+The main way, is the delay within pinging messages to a server and producing an output result.
+This applies to wide variety of scenarios (evaluating code, network communication, including osc and midi, etc.)
```
Comment (vivid-synth) [inline HelpSource/Classes/Server.schelp]: Maybe we should link to a discussion of server latency instead?

**Comment by brianlheim** [comment]
Closing, see #2645.

**Pull request closed without merging**